src/main/scala/org/broadinstitute/dsde/vault/services/uBAM/DescribeServiceHandler.scala

```diff
@@ -37,7 +37,7 @@ case class DescribeServiceHandler(requestContext: RequestContext, version: Int,
       val redirects = resolvedUBam.files.map {
         case (fileType, _) => (fileType, VaultConfig.Vault.ubamRedirectUrl(resolvedUBam.id, fileType))
       }
-      requestContext.complete(UBam(resolvedUBam.id, redirects, resolvedUBam.metadata).toJson.prettyPrint)
+      requestContext.complete(UBam(resolvedUBam.id, redirects, resolvedUBam.metadata, resolvedUBam.properties).toJson.prettyPrint)
```

Comment:
Since we're copying the incoming UBam except for one modification, replacing the new `UBam()` with `resolvedUBam.copy(files = redirects)` would make that more explicit.